code/modules/antagonists/slaughter/slaughter.dm

```diff
@@ -1,5 +1,5 @@
 //////////////////The Monster
-
+/* YOGS - !!!MOVED TO yogstation/code/modules/antagonists/slaughter/slaughter.dm!!!
```

Comment:
Don't comment this out, just remove this from the .dme if you don't need it. (also move this to line 1 if you feel like it)